### ISSUE TITLE
App doesn’t work with latest core master

### ISSUE THREAD

**Issue by jancborchardt** [simeonackermann/OC-User-Conversations#37]
Hey @simeonackermann – I wanted to test out (and maybe contribute) to your Chat app again. :)

However after installing it, when I try to open the app I am faced with a white page and error:
```
    User
    Fatal error: Class 'OC_Avatar' not found in /home/jan/owncloud/apps/conversations/lib/conversations.php on line 458
```

owncloud.log:
```
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#286","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#288","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#286","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#288","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#286","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#288","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#286","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Undefined index: rooms at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#288","level":0,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
{"reqId":"JOHg\/32gceGFWmkr\/hRl","remoteAddr":"127.0.0.1","app":"PHP","message":"Class 'OC_Avatar' not found at \/home\/jan\/owncloud\/apps\/conversations\/lib\/conversations.php#458","level":3,"time":"2015-04-17T12:30:51+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/conversations\/"}
```

**Comment by jancborchardt** [comment]
And btw – the
`Debug	PHP	Undefined index: rooms at /home/jan/owncloud/apps/conversations/lib/conversations.php#286`
gets spammed into the log to no end. ;)

**Comment by simeonackermann** [comment]
Thanks for testing and reporting!
This issue should by fixed with last commit de004ffb7969f23464aae8efb3193f6ea5eed88c

**Comment by jancborchardt** [comment]
Yep, fixed. Thank you! :)
